examples/workflow/pygrb/pygrb.ini

```diff
@@ -37,8 +37,8 @@ segments-L1-science-name = L1:DMT-SCIENCE:4
 ;segments-V1-science-name = V1:ITF_SCIENCEMODE:6
 segments-database-url = https://segdb.ligo.caltech.edu
 segments-veto-definer-url = https://www.lsc-group.phys.uwm.edu/ligovirgo/cbc/public/segments/S6/H1L1V1-S6_CBC_LOWMASS_D_OFFLINE-961545543-0.xml
-segments-veto-categories = 2,3,4
-segments-minimum-segment-length = 2000
+segments-maximum-veto-category = 3
```

Comment:
I think this won't work, I think the new way of specifying this in the segments module is segments-veto-categories = 2,3 as seen above. Better to specify all used segments rather than assume they are cumulative.